packages/core/README.md

```diff
@@ -294,12 +295,21 @@ The `clear-text` reporter supports three additional config options:
 
 The `dashboard` reporter sends a report to https://dashboard.stryker-mutator.io, enabling you to add a mutation score badge to your readme, as well as hosting your html report on the dashboard. It uses the [dashboard.*](#dashboard) configuration options. See [the Stryker handbook](https://github.com/stryker-mutator/stryker-handbook/blob/master/dashboard.md) for more info.
 
-<a name="sandbox.fileHeaders"></a>
-### `sandbox.fileHeaders` [`object`]
+<a name="sandbox.ignorePatterns"></a>
+### `sandbox.ignorePatterns` [`string[]`]
 
-Default: `{ "**/*+(.js|.ts|.cjs|.mjs)?(x)": "/* eslint-disable */\n// @ts-nocheck\n" }`
+Default: `['**/__tests__/**/*.{ts,tsx}', '**/*.{spec,test}.{ts,tsx}', '**/*{Spec,Test}.{ts,tsx}']`
```

Comment:
Ignoring the test files will not help in scenario's where typescript is used.

For example:

```js
// concat.ts
function concat(a: string, b: string) {
   return a + b; // return type of this function will change because of mutation switching
}

// __tests__/concat.spec.ts
it('should concat', () => {
  const actual: string = concat('a', 'b');
  expect(actual).eq('ab');
});
```

In this case, `__tests__/concat.spec.ts` will give a compile error. You would notice it if your using `buildCommand: "tsc -b"` or you use `jest-ts` for example.